symposion/reviews/views.py

```diff
@@ -263,6 +289,7 @@ def review_detail(request, pk):
 
                 tags = proposal_tags_form.cleaned_data['tags']
                 proposal.tags.set(*tags)
+                proposal.cache_tags()
```

Comment:
I kind of wish that this were in a signal. Maybe the `save()` customization as well. What do you think?